lint.whitelist

```diff
@@ -326,6 +326,7 @@ OPEN-NO-MODE: css/*
 PRINT STATEMENT: css/*
 CONTENT-VISUAL: css/*
 CONTENT-MANUAL: css/*
+CONTENT-VISUAL: payment-request/PaymentMethodChangeEvent/methodName-attribute.https.html
```

Comment:
This seems wrong. Why is it getting picked up as a visual test?

Reply:
agree... filed #11479 because this is not documented.

Reply:
(reverting this, btw)